src/ui/components/DropdownMenu/styles.scss

```diff
@@ -0,0 +1,88 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+.DropdownMenu {
+  @include text-align-start();
+
+  display: inline-block;
+  padding-bottom: 6px;
+  padding-top: 6px;
+  position: relative;
+}
+
+.DropdownMenu-text {
+  @include font-regular();
+
+  color: $white;
+  font-size: $font-size-s;
+  font-weight: normal;
+  text-decoration: none;
+
+  .Icon-inverted-caret {
+    @include margin-start(5px);
+
+    height: 10px;
+    opacity: 0.6;
+    transition: opacity $transition-short ease-in-out;
+    width: 9px;
+  }
+}
+
+.DropdownMenu--active {
```

Comment:
Here's a weird thought: we should target, as the root selector, the element we want to style and if there are contextual styles applied to it from a state like hover/active, they should be inside the selector. So instead of a section of restyles based on `.DropdownMenu--active {}` being set you'd do:

```scss
.DropdownMenu-items {
  [all the normal styles here]

  .DropdownMenu--active & {
    display: block;
  }
}
```

I've tried to do that (though maybe not at first, but tried going forward about three months ago) and I found it works well and makes the file easier to understand. I thought I said so in the CSS guides but forget now.

Anyway, just a thought!

Reply:
I like it!